bridge/bridge/ethereum/EthereumConnector.py

```diff
@@ -253,6 +266,8 @@ async def announce_transaction(self, transaction_payload):
 	async def try_wait_for_announced_transaction(self, transaction_hash, desired_status, timeout_settings):
 		"""Tries to wait for a previously announced transaction to transition to a desired status."""
 
+		await self._require_synced('check transaction status')
```

Comment:
Is this due to the node rollback?

Reply:
this was recommended because if node restarts it will be in a syncing state until it catches up